test/season_stats_test.rb

```diff
@@ -37,10 +37,9 @@ def test_it_returns_name_of_team_with_fewest_tackles
     assert_equal "Atlanta United", @season_stats.fewest_tackles("20132014")
   end
 
-  def test_it_can_return_most_accurate_team
-    skip
-    assert_equal "the", @season_stats.most_accurate_team(20152016)
-  end
+  # def test_it_can_return_most_accurate_team
+  #   assert_equal "the", @season_stats.most_accurate_team(20152016)
```

Comment:
Interesting assert_equal here. Can you clarify for me?

Reply:
It is a placeholder to run test